### PR TITLE
Development Of new Feature Assigned to me is completed.

### PR DESCRIPTION
**Pull request by pavankumart46**
Kindly Check the code and give me reviews

### CODE REVIEW

File: cse6.txt [hunk 1]
```diff
@@ -1 +1 @@
-This file is created on the new branch "Feature". What ever the changes that i make on this new branch, if the code is compliant with the code on master branch, My Project manager of team lead will integrate the changes back to the master branch.
```
Comment (harideep8) [inline cse6.txt]: These lines are important. Why did you remove a this.

**Comment by 174G1A0558** [comment]
harideep sir
